src/main/features/eligibility/views/macro/mcolSupport.njk

```diff
@@ -0,0 +1,10 @@
+{% from "callCharges.njk" import callCharges %}
+
+{% macro mcolSupportWidget(label = 'Help and advice') %}
```

Comment:
the two support macros should live next to each other in the same file I think
Check if legal uses the other one and if it doesn't you should be able to move it here